roles/install/README.md

```diff
@@ -83,4 +83,4 @@ MIT
 
 ## Author Information
 
-Tom Page
+Redhat-Cop.
```

Comment:
Sorry, I meant change the author to not be me for the kerberos role. This role can stay as me as I did contribute it!